### PR TITLE
My GSOD participation Blog

### PR DESCRIPTION
**Pull request by vasvi-sood**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: GSoD Applicant (How-to guides) 
    author: https://github.com/vasvi-sood/
    categories: Applicayion blog
    tags:
      - GSod 2021
      
    image: /assets/2021/06-Vasvi-applicant-for-technical-writer-gsod/my photo.png
    ---

    One line bio

 Vasvi Sood, NIT Hamirpur

### CODE REVIEW

**Comment by vasvi-sood** [comment]
Could you please tell me the reason why the test failed?

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@vasvi-sood Tests are failing because you have not formatted the markdown file properly. There is a necessary line break and indentations. You can format the markdown file using [Prettier](https://prettier.io).

File: jekyll/_posts/2021-05-06-Vasvi-applicant-for-technical-writer-gsod.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (iamrajiv) [inline jekyll/_posts/2021-05-06-Vasvi-applicant-for-technical-writer-gsod.md:10]: @vasvi-sood Add Blog Header, including title, author, image, categories, tags, etc.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2021-05-06-Vasvi-applicant-for-technical-writer-gsod.md (hunk 1 above)
Comment (vasvi-sood) [reply jekyll/_posts/2021-05-06-Vasvi-applicant-for-technical-writer-gsod.md:10]: Okay thanks, I'll submit a new pull request with the required changes.

**Pull request closed without merging**